docs/topics/signals.rst

```diff
@@ -16,6 +16,54 @@ deliver the arguments that the handler receives.
 You can connect to signals (or send your own) through the
 :ref:`topics-api-signals`.
 
+Here is a simple example showing how you can catch signals and perform some action.
+
+    from scrapy.crawler import Crawler, CrawlerProcess
+    from scrapy.utils.project import get_project_settings
+    from scrapy import signals
+
+    from dirbot.spiders import dmoz
+
+    #Initiating a crawler object with the dmoz spider
+    crawler = Crawler(dmoz.DmozSpider, get_project_settings())
+
+    #Connecting callback functions to signals
+    def connect_callbacks_to_signals(crawler):
+      crawler.signals.connect(engine_started, signal = signals.engine_started)
+      crawler.signals.connect(spider_opened, signal = signals.spider_opened)
+      crawler.signals.connect(spider_closed, signal = signals.spider_closed)
+      crawler.signals.connect(engine_stopped, signal = signals.engine_stopped)
+      crawler.signals.connect(item_scraped, signal = signals.item_scraped)
+
+
+    def engine_started():
+        print "###Engine started###"
+
+
+    def spider_opened():
```

Comment:
Don't forget the [signature](http://doc.scrapy.org/en/latest/topics/signals.html#scrapy.signals.spider_closed) here (`def spider_opened(spider, reason):`), also for all following callbacks.